### ISSUE TITLE
Dependency conflict

### ISSUE THREAD

**Issue by chpl** [env0/checkov-plugin#5]
Customer is trying to use the plugin but gets following error
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
awscli 1.29.25 requires botocore==1.31.25, but you have botocore 1.34.25 which is incompatible.
awscli 1.29.25 requires s3transfer<0.7.0,>=0.6.0, but you have s3transfer 0.10.2 which is incompatible.
```

**Comment by chpl** [comment]
fixed in #6

**Comment by chpl** [comment]
fixed in #6
